src/main/java/br/com/openenade/api/nota/Avaliacao.java

```diff
@@ -0,0 +1,131 @@
+package br.com.openenade.api.nota;
+
+import java.io.Serializable;
+
+public class Avaliacao implements Serializable {
+
+    /**
+     * 
+     */
+    private static final long serialVersionUID = 1L;
+
+    private int concluintesInscritos;
+    private int concluintesParticipantes;
+    private double notaBrutaFG;
+    private double notaPadronizadaFG;
+    private double notaBrutaCE;
+    private double notaPadronizadaCE;
+    private double enadeContinuo;
+    private int enadeFaixa;
+
+    @Override
+    public int hashCode() {
```

Comment:
No need for hashCode and equals methods in Avaliacao.

Reply:
Ok o/